_site/assets/fonts/courier-regular/readme/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta name="viewport" content="width=device-width" />
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<meta http-equiv="refresh" content="5;url=http://fontsgeek.com?ref=readme">
+<title>Fontsgeek</title>
+<style>
+/* ------------------------------------- 
```

Comment:
Can you remove this file?
This file is in `_site` folder. The `_site` folder is generated after building and should be excluded.
If we need this file it should be in `assets/fonts` folder and not `_site/assets/fonts` folder

Reply:
@zabil - I am going to take a stab at this since @streisguth is on holiday.

Reply:
@zabil - no luck for me :(. I implemented your suggestions the failure seems to persist.